### PR TITLE
Milestone1 issues other

### LINKED ISSUE #31: remove unused files
remove the unused file app/Helpers/ActiveDirectory.php

### LINKED ISSUE #33: Move user email addresses out of app config and into env
CD admin should have super admin rights.

login without simple saml all permisions and view telescope permissions.

set users in a CSV list in .env file

_Originally posted by @woodseowl in https://github.com/CU-CommunityApps/cd-son/pull/25_

### LINKED ISSUE #35: remove pantheon reference from simplesaml config
https://github.com/CU-CommunityApps/cd-son/blob/5d20d77d030a752f2bc4d6c1800de3d1ca48fd20/private/simplesaml/config/authsources.php#L3-L5

Remove the reference to pantheon server and use laravel ENV

### PR DESCRIPTION
**Pull request by philwilliammee**
closes #33
closes #35
closes #31

file clean up, removes pantheon reference, set CD_ADMIN variable in .env

### CODE REVIEW

File: app/Providers/TelescopeServiceProvider.php [hunk 1]
```diff
@@ -63,10 +63,8 @@ protected function hideSensitiveRequestDetails()
     protected function gate()
     {
         Gate::define('viewTelescope', function ($user) {
-            return in_array($user->email, [
-                'psw58@cornell.edu',
-                'elw234@cornell.edu',
-            ]);
+            $cd_admins = explode(',', env('CD_ADMINS'));
```
Comment (woodseowl) [inline app/Providers/TelescopeServiceProvider.php:66]: Just allows a little more robustness in the ENV file...
```suggestion
            $cd_admins = preg_split('/[, ]+/', env('CD_ADMINS'));
```

File: private/simplesaml/config/authsources.php [hunk 2]
```diff
@@ -1,8 +1,8 @@
 <?php
 
-$idp = (isset($_SERVER['PANTHEON_ENVIRONMENT']) && $_SERVER['PANTHEON_ENVIRONMENT'] == 'live')
-  ? 'https://shibidp.cit.cornell.edu/idp/shibboleth'
-  : 'https://shibidp-test.cit.cornell.edu/idp/shibboleth';
+$idp = (preg_match('(test|dev|localhost|lando|lndo)', $_SERVER['SERVER_NAME']) === 1)
+    ? 'https://shibidp-test.cit.cornell.edu/idp/shibboleth'
+    : 'https://shibidp.cit.cornell.edu/idp/shibboleth';
```
Comment (woodseowl) [inline private/simplesaml/config/authsources.php:5]: We already have access to `$_ENV` thanks to simplesaml/config/config.php, so I think we can just use that and keep the "only if" quality the original?
```suggestion
$idp = ($_ENV['APP_ENV'] == 'live')
    ? 'https://shibidp.cit.cornell.edu/idp/shibboleth'
    : 'https://shibidp-test.cit.cornell.edu/idp/shibboleth';
```

File: .env.example [hunk 3]
```diff
@@ -44,3 +44,5 @@ PUSHER_APP_CLUSTER=mt1
 
 MIX_PUSHER_APP_KEY="${PUSHER_APP_KEY}"
 MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
+#a comma seperated list of netids. No whitespaces!
```
Comment (woodseowl) [inline .env.example:47]: See below... made it so it's not quite as strict. :)
```suggestion
# A comma separated list of netids.
```

**Comment by philwilliammee** [comment]
yep all suggestions are good thank you.

**Pull request merged**